{"name": "Проверка поля 'id' в успешном ответе", "status": "passed", "description": "Проверка, что при успешной авторизации в ответе есть 'id'", "start": 1752323998398, "stop": 1752323998630, "uuid": "979edef6-9bd8-4b71-995b-92c76512271d", "historyId": "c2d8aef6f83994a597fe35cee9c8ff31", "testCaseId": "c2d8aef6f83994a597fe35cee9c8ff31", "fullName": "Tests.test_login_courier.TestLoginCourier#test_login_response_text", "labels": [{"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-T2S04D8"}, {"name": "thread", "value": "80-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_login_courier"}]}